Lib/traceback.py

```diff
@@ -618,7 +626,7 @@ class TracebackException:
 
     def __init__(self, exc_type, exc_value, exc_traceback, *, limit=None,
             lookup_lines=True, capture_locals=False, compact=False,
-            _seen=None):
+            stack_summary_cls=None, _seen=None):
```

Comment:
This seems like a good approach to customizing the behavior while reusing most of the exception printing code. There's precedent for this style of expansion with `JSONEncoder` and `cls` in the `json` module. Any thoughts @terryjreedy?